app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    list_of_people = []
+
+    for data in people:
+        person_inst = Person(data["name"], data["age"])
+        list_of_people.append(person_inst)
+
+    for data in people:
+        for names in Person.people.keys():
+            if "wife" in data and data["wife"] == names:
```

Comment:
Use data.get("wife") instead of two conditions

Reply:
Renamed  list_of_people, used .get to look through keys, replaced nested loop